<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color= #353030;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>160</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plot_selections">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QRadioButton" name="graph_amps">
       <property name="text">
        <string>CURRENT (A)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="graph_volt">
       <property name="text">
        <string>VOLTAGE (V)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="graph_TC1">
       <property name="text">
        <string>TC1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="graph_power">
       <property name="text">
        <string>POWER (P)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="selectable_variables">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="select_amps">
       <property name="text">
        <string>CURRENT (A)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_volt">
       <property name="text">
        <string>VOLTAGE (V)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_tc1">
       <property name="text">
        <string>TC1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_power">
       <property name="text">
        <string>POWER (P)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollBar" name="move_across_graph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="zoomer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_calculated_graph">
    <property name="title">
     <string>Add calculated graph</string>
    </property>
   </widget>
   <widget class="QMenu" name="menucalculate_description">
    <property name="title">
     <string>calculate description</string>
    </property>
   </widget>
   <addaction name="menuAdd_calculated_graph"/>
   <addaction name="menucalculate_description"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
